src/pages/Notice/NoticeDetail/components/NoticeContent/NoticeContent.tsx

```
@@ -26,16 +26,14 @@ export const NoticeContent = ({ notice }: NoticeContentProps) => {
 
       <div className={styles["notice-content"]}>
-        {notice.link ? (
-          <iframe
-            src={notice.link}
-            className={styles["notice-iframe"]}
-            title="공지사항 원문"
-            sandbox="allow-same-origin allow-scripts allow-popups allow-forms"
+        {notice.content ? (
+          <div
+            className={styles["notice-html-content"]}
+            dangerouslySetInnerHTML={{ __html: notice.content }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)